tests/platform/cloud/AcceptanceTests_oauth2_webPrefsConsumer.txt

```diff
@@ -0,0 +1,7 @@
+AcceptanceTests_webPrefsConsumer.js
+
+Description:
+This is a series of tests using the online flowmanager to test the workflow of that web preferences consumers to request access tokens that are then used to request filtered user settings from GPII Cloud using OAuth2 authorization code grant. 
```

Comment:
We can remove the "that" in "test the workflow of that web preferences consumers to".